File: Server/src/common/infrastructure/UserStatistics.cpp
#include "UserStatistics.h"

UserStatistics::UserStatistics(const int points, const int gamesPlayed, const int questionsAnswered,
                               const int questionsAnsweredCorrect, const int timeOnQuestionsOverall,
                               const int timePerQuestionsAvg)
    : points(points), gamesPlayed(gamesPlayed), questionsAnswered(questionsAnswered),
      questionsAnsweredCorrect(questionsAnsweredCorrect), timeOnQuestionsOverall(timeOnQuestionsOverall),
      timePerQuestionsAvg(timePerQuestionsAvg)
{}

File: Client/Trivia/Models/User/UserStatistics.cs
namespace Trivia.Models.User;

public class UserStatistics
{
    public required int Points { get; init; }
    public required int GamesPlayed { get; init; }
    public required int QuestionsAnswered { get; init; }
    public required int QuestionsAnsweredCorrectly { get; init; }
    public required int TimeOnQuestionOverall { get; init; }
    public required int TimeOnQuestionsAvg { get; init; }
}